src/App.tsx

```diff
@@ -266,7 +266,9 @@ const Main = ({ children }: { children: ReactNode }) => {
 
   return (
     <RootSiblingParent>
-      <SafeAreaProvider>{children}</SafeAreaProvider>
+      <SafeAreaProvider {...props} style={{ backgroundColor: 'black' }}>
```

Comment:
To make sure that the props.styles does not get overriden by `style={{ backgroundColor: 'black' }}>` we need to do something like:

```tsx
const Main = ({ children, styles, ...props }: ViewProps) => {
...
      <SafeAreaProvider {...props} style={[{ backgroundColor: 'black' }, styles]}>
```

Reply:
It's on purpose. I want to make sure that the background is black.